Networking/NetworkingTests/Mapper/OrderNotesMapperTests.swift

```diff
@@ -62,12 +87,18 @@ private extension OrderNotesMapperTests {
         return try! OrderNotesMapper().map(response: response)
     }
 
-    /// Returns the OrderNotesMapper output upon receiving `orders-load-all`
+    /// Returns the OrderNotesMapper output upon receiving `order-notes`
     ///
     func mapLoadAllOrderNotesResponse() -> [OrderNote] {
         return mapNotes(from: "order-notes")
     }
 
+    /// Returns the OrderNotesMapper output upon receiving `order-notes-without-data`
```

Comment:
super nit:
```suggestion
    /// Returns the [OrderNote] output upon receiving `order-notes-without-data`
```

Reply:
Updated in f0f29442f88208e9a99e086b90d9f1ba8385cd6c.